desktop/apps/artist/client/views/overview.coffee

```diff
@@ -41,9 +41,9 @@ module.exports = class OverviewView extends Backbone.View
       heightBreakOffset: 20
       onExpand: =>
         if @useNewArtworkFilter
-          @filterView.sticky.rebuild()
+          @filterView.sticky?.rebuild()
         else
-          @sticky.rebuild()
+          @sticky?.rebuild()
```

Comment:
Fixes https://sentry.io/artsynet/force-staging/issues/445093968/

Reply:
Ah I was just looking at this too - weirdly this file doesn't even define `@sticky` so this could probably use a #FIXME comment 😄